themes/default/index.twig

```diff
@@ -1,49 +1,70 @@
 <!DOCTYPE html>
-<html lang="en" class="no-js">
+<html lang="en">
 <head>
-    <meta charset="utf-8" />
-
-    <title>{% if meta.title %}{{ meta.title }} | {% endif %}{{ site_title }}</title>
+    <!-- Page Meta -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
     {% if meta.description %}
-        <meta name="description" content="{{ meta.description|striptags }}">
+    <meta name="description" content="{{ meta.description|striptags }}">
     {% endif %}{% if meta.robots %}
-        <meta name="robots" content="{{ meta.robots }}">
+    <meta name="robots" content="{{ meta.robots }}">
     {% endif %}
 
-    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700" type="text/css" />
-    <link rel="stylesheet" href="{{ theme_url }}/style.css" type="text/css" />
+    <!-- Page Title -->
+    <title>{% if meta.title %}{{ meta.title }} | {% endif %}{{ site_title }}</title>
+
+    <!-- Fonts -->
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700" type="text/css">
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="{{ base_url }}/vendor/twbs/bootstrap/dist/css/bootstrap.min.css" type="text/css" />
```

Comment:
I've found a [good article about this problem](http://www.nikosams.net/blog/17_composer_npm_bower_assets_installation_using_composer-extra-assets) elaborating our possibilities. Additionally we can:
* write a simple helper script to copy the bootstrap assets to the desired directory
* add a whitelist entry to `.htaccess` (this will prevent users (like me) to put the vendor directory somewhere outside `httpdocs`)
* use one of Bootstraps CDNs

So we have at least 10 possibilites. Any thoughts @theshka, @dav-m85?